docs/src/components/MarkdownPage/AdditionalLinks.js

```diff
@@ -203,7 +203,11 @@ const AdditionalLinks = (props: Props) => {
           secondary
           blank
           className={getClassName('bpkdocs-markdown-page__edit-button')}
-          href={`https://github.com/skyscanner/backpack-docs/edit/master/docs/src/static-pages/${fileName}`}
+          href={
+            platform
+              ? `https://github.com/skyscanner/backpack-docs/edit/master/docs/src/pages/components/${fileName}`
```

Comment:
If/when we move the mdx files to live within their respective repos, this logic will become more complex by looking at the platform, but yagni for now